### ISSUE TITLE
Add a way to configure threshold for OpenURI portal

### ISSUE THREAD

**Issue by mariospr** [flatpak/xdg-desktop-portal#32]
As mentioned in https://github.com/flatpak/xdg-desktop-portal/issues/30#issuecomment-228920715, it would be good to have a way to configure this threshold (and probably other portal-specific options that we might need) so that it can be adapted to different needs.

I initially thought of a key file under `/etc`, but perhaps GSettings is just fine, and cleaner/easier.

**Comment by matthiasclasen** [comment]
yet another alternative is to store the threshold in the permissions db, together with the count, per application.

**Comment by mariospr** [comment]
I suppose, so, but the permissions store is harder to manipulate than a configuration setting and makes it harder to set different options per-deployment, so I'm not sure.

I think GSettings or a key file would be a better way to do this if setting options per application was not a requirement. If it is, perhaps extending the command line options of flatpak to accomodate portal-specific opttions might work, as those could be specified in the json file for the builder too.

What do you think?

**Comment by matthiasclasen** [comment]
Commandline options and metadata is not sufficient.

We need to give users control over this, in the end, with a control-center UI. Looking at the code for the privacy panel, it already reads geolocation settings out of the permission store; so that seems no problem on the control-center side.

**Comment by mariospr** [comment]
Understood, permissions store it is then. I'll try to find some time tomorrow, otherwise next week. Thanks!

**Comment by mariospr** [comment]
I have to apologize for the long time taken to get back to this, but I finally got to do it today. See the pull request here: https://github.com/flatpak/xdg-desktop-portal/pull/56

**Comment by mariospr** [comment]
While running my flatpak'ed version of Thunderbird today, I noticed I made a mistake in the main patch merged last week: when clicking on links, Thunderbird only showed the app chooser dialog once because, after the first time, the threshold was incorrectly set to zero in the permissions store.

The problem is that, while I set sane defaults in `parse_permissions()` thinking of this use case, the truth is that `get_latest_choice_info()` would not even call that if the app was not already registered in the permissions store, and so the defaults set (i.e. zero) by the callers of `get_latest_choice_info()` would prevail, messing everything up.

I've pushed an additional commit in https://github.com/flatpak/xdg-desktop-portal/pull/60 associated to this same issue, but let me know if you'd rather have me open a new one instead.

Sorry for the silly mistake
